projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -474,109 +483,110 @@ function wpcom_display_global_styles_launch_bar( $bar_controls ) {
+	?>
+
+	<div class="launch-banner" id="launch-banner" style="display:none;">
+		<div class="launch-banner-content">
+			<div class="launch-banner-section bar-controls">
+				<div class="launch-bar-global-styles-button">
+					<?php if ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) : // Workaround for the shadow DOM used on Atomic sites. ?>
+						<style id="wpcom-launch-bar-global-styles-button-style">
+							<?php include __DIR__ . '/dist/wpcom-global-styles-view.css'; ?>
+							.hidden { display: none; }
+						</style>
+						<script id="wpcom-launch-bar-global-styles-button-script">
+							const launchBarUserData = {
+								blogId: <?php echo method_exists( '\Jetpack_Options', 'get_option' ) ? (int) \Jetpack_Options::get_option( 'id' ) : get_current_blog_id(); ?>,
+								isAtomic: true,
+							};
+							<?php
+							include __DIR__ . '/dist/wpcom-global-styles-view.min.js';
+							$asset_file   = plugin_dir_path( __FILE__ ) . 'dist/wpcom-global-styles-view.asset.php';
+							$asset        = file_exists( $asset_file ) ? require $asset_file : null;
+							$dependencies = $asset['dependencies'] ?? array();
+							foreach ( $dependencies as $dep ) {
+								$dep_script = wp_scripts()->registered[ $dep ];
+								if ( ! $dep_script ) {
+									continue;
+								}
+								include ABSPATH . $dep_script->src;
+							}
+							?>
+						</script>
+					<?php endif; ?>
+					<div class="launch-bar-global-styles-popover hidden">
+						<div class="launch-bar-global-styles-close">
+							<svg xmlns="http://www.w3.org/2000/svg" height="48" viewBox="0 96 960 960" width="48"><path d="m249 849-42-42 231-231-231-231 42-42 231 231 231-231 42 42-231 231 231 231-42 42-231-231-231 231Z"/></svg>
+						</div>
+						<div class="launch-bar-global-styles-message">
+							<?php
+							$support_url = function_exists( 'localized_wpcom_url' )
+								? localized_wpcom_url( 'https://wordpress.com/support/using-styles/' )
+								// phpcs:ignore WPCOM.I18nRules.LocalizedUrl.UnlocalizedUrl
+								: 'https://wordpress.com/support/using-styles/';
+
+							$message = sprintf(
+								/* translators: %1$s - documentation URL, %2$s - the name of the required plan */
+								__(
+									'Your site includes <a href="%1$s" target="_blank">premium styles</a> that are only visible to visitors after upgrading to the %2$s plan or higher.',
+									'jetpack-mu-wpcom'
 								),
-							)
-						)
-					);
-					?>
+								$support_url,
+								get_store_product( $gs_upgrade_plan )->product_name
+							);
+							printf(
+								wp_kses(
+									$message,
+									array(
+										'a' => array(
+											'href'   => array(),
+											'target' => array(),
+										),
+									)
+								)
+							);
+							?>
+						</div>
+						<a
+							class="launch-bar-global-styles-upgrade"
+							href="<?php echo esc_url( $upgrade_url ); ?>"
+						>
+							<?php echo esc_html__( 'Upgrade now (updated)', 'jetpack-mu-wpcom' ); ?>
```

Comment:
I guess this string change is unintended? 
```suggestion
							<?php echo esc_html__( 'Upgrade now', 'jetpack-mu-wpcom' ); ?>
```

Reply:
Good catch, that was a debug change indeed.